app/src/components/common/outcome_table/index.tsx

```diff
@@ -115,7 +111,7 @@ export const OutcomeTable = (props: Props) => {
             <BarDiagram
               outcomeName={outcomeName}
               isWinning={isWinning}
-              probability={probability}
+              probability={+probability.toFixed(2)}
```

Comment:
Mmmm without the method `toFixed`, looks like this...
![Screenshot_20200106_164654](https://user-images.githubusercontent.com/1144028/71844201-6fff6600-30a4-11ea-807f-c86b57022bc4.png)

Reply:
It seems to me that `BarDiagram` should be responsible of trimming the number of digits.